docs/licensing/chibios.md

```diff
@@ -3,42 +3,42 @@
 
 **About this document**
 
-This document provides a general overview on the licensing options available when using [ChibiOS](http://chibios.org) as **nanoFramework** RTOS component.
-We've been in touch with the ChibiOS sales team to clarify this as much as possible.
+This document provides a general overview of the licensing options available when using [ChibiOS](http://chibios.org) as the **nanoFramework** RTOS component.
+The nanoFramework team have been in contact with the ChibiOS sales team to ensure accuracy of the following information at time of print (October 2017).
 
-For details and to discuss your particular situation we strongly recommend getting in touch with the ChibiOS sales [here](http://chibios.org/dokuwiki/doku.php?id=chibios:licensing:quote).
+For details or to discuss your particular situation, we strongly recommend getting in touch with the ChibiOS [sales team](http://chibios.org/dokuwiki/doku.php?id=chibios:licensing:quote).
 
 
-## What exactly is being used from ChibiOS in **nanoFramework**?
+## What exactly is being used from ChibiOS in **nanoFramework firmware**?
 
-**nanoFramework** is built against _unmodified_ ChibiOS sources. It's using:
-* The [HAL](http://chibios.org/dokuwiki/doku.php?id=chibios:product:hal:start), which is released under Apache License 2.0 meaning that it's 100% free also for commercial use.
-* The [RT](http://chibios.org/dokuwiki/doku.php?id=chibios:product:rt:start), which is released under GPL3. See bellow about the licensing options according to your use. 
+**nanoFramework** is built against _unmodified_ ChibiOS sources via a mirror. It's using:
```

Comment:
I don't think that adding this information about the way the source code gets into the build is relevant.
Actually (with folks that are more picky) can lead to unfertile discussions about the source code origin...
But, this is not critical.

Reply:
The only reason I mentioned mirror is because it is not directly from there official repository which can lead to confusion in my opinion. Happy to change back if you want... let me know.

Reply:
I guess this won't hurt. Let it be with the mirror mention.